src/components/CartProductsListItem/CartProductsListItem.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import './CartProductsListItem.scss';
+
+const CartProductsListItem = () => {
+  const [number, setNumber] = useState(1);
+
+  const subtract = () => {
+    setNumber(number - 1);
+    if (number < 2) {
+      setNumber(1);
+    }
+  };
+
+  return (
+    <li className="cartProductsListItem">
+      <div className="cartProductstName">로즈티</div>
+      <div className="cartProductsAmount">15g</div>
+      <div className="cartProductsCount">
+        <button className="minusButton" onClick={subtract}>
+          ➖
+        </button>
+        <span className="cartProductstNum">{number}</span>
+        <button
+          className="plusButton"
+          onClick={() => {
+            setNumber(number + 1);
+          }}
+        >
```

Comment:
```suggestion
        <button className="minusButton" onClick={()=>{handleItemNum(-1}}>
          ➖
        </button>
        <span className="cartProductstNum">{number}</span>
        <button
          className="plusButton"
          onClick={() => {
            handleItemNum(1);
          }}
        >
```

위의 리뷰를 적용한다면 아래 코드도 이렇게 변할 수 있겠네요!